### ISSUE TITLE
Is it possible to use this module with Angular using ngx-quill ? 

### ISSUE THREAD

**Issue by ayyazzafar** [afry/quill-mention#153]
First of all I want to thank you for building such a nice extension. 
We are currently using Angular and wanted to use quill in angular.
There is a very nice module ngx-quill that allows you to use quill in agnular. 

But I am not sure if it is easily possible to use quill-mention with that angular plugin or not ? 

Can you guide if that is possible ?
Thanks

**Comment by Vbboi24** [comment]
Yes, it is possible. check out https://github.com/KillerCodeMonkey/ngx-quill-example

There's an example Available
